bootstrapper/internal/journald/journald.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+/*
+Package journald provides functions to read and collect journald logs.
+*/
+package journald
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os/exec"
+)
+
+type command interface {
+	Output() ([]byte, error)
+}
+
+// Collector represents a command that is executed by journalctl.
+type Collector struct {
+	cmd command
+}
+
+// NewCollector creates a new JournaldCommand.
+func NewCollector(ctx context.Context, service string) (*Collector, error) {
+	cmd := exec.CommandContext(ctx, "journalctl", "-u", service)
```

Comment:
Wasn't the plan to get all logs, to keep it simple?

Reply:
I thought that it would be better to have a more flexible package that can read logs from any service if we require logs from other services one day. In the bootstrapper, this will be only used to read logs from the bootstrapper and the kubelet service for now.

Reply:
As said before, imo we should just collect all units right away if init fails. How are you going to store the logs of the units you want to collect on the CLI side? In multiple files? Just concatenate the strings?

Reply:
I thought of using multiple files in the form of `error-{SERVICE_NAME}.log`. But I believe I understand your point, if you want me to collect and stream all systemd logs then I can change the code accordingly.

Reply:
Multiple files will likely make it harder to request that data from the user.